capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -6,28 +6,55 @@ import { act } from "react-dom/test-utils";
 import BookSearchTile from "./components/BookSearchTile.jsx";
 import App from "./App.jsx";
 import {
-  Route, 
-  BrowserRouter as Router, 
+  Route,
+  BrowserRouter as Router,
   Link
-} from 'react-router-dom'; 
+} from 'react-router-dom';
 
 const container = document.createElement("div");
 document.body.appendChild(container);
 const book = {
   title: "Title",
-  author: ["Author"],
+  authors: ["Author"],
   thumbnailLink: "https://http.cat/100.jpg"
 };
+const getUserBookLists = () => {
+  let userBookLists = [{ id: "1", name: "Best Books" },
```

Comment:
Is this test separate from the main Component? Will the test break when I merge my PR?

Reply:
Yeah, these tests are completely separate from the BookSearchList component. In this test, I just have these functions to make it easier to generate consistent test data.